tests/regression/splash/conftest.py

```diff
@@ -2,10 +2,38 @@
 
 from importlib import resources
 
-import numpy as np
+import pandas as pd
 import pytest
 import xarray
 
+EXPECTED_NAME_MAP: dict[str, str] = {
+    "my_nu": "nu",
+    "my_lambda": "lambda_",
+    "dr": "distance_factor",
+    "delta": "declination",
+    "hs": "sunset_hour_angle",
+    "ra_d": "daily_solar_radiation",
+    "tau": "transmissivity",
+    "ppfd_d": "daily_ppfd",
+    "rnl": "net_longwave_radiation",
+    "hn": "crossover_hour_angle",
+    "rn_d": "daytime_net_radiation",
+    "rnn_d": "nighttime_net_radiation",
+    "sat": "sat",
+    "lv": "lv",
+    "pw": "pw",
+    "psy": "psy",
+    "econ": "econ",
+    "rx": "rx",
+    "hi": "hi",
+    "cond": "cond",
+    "eet_d": "eet_d",
+    "pet_d": "pet_d",
+    "aet_d": "aet_d",
+    "wn": "wn",
+    "ro": "ro",
```

Comment:
Should these also be changed to something more understandable?

Reply:
Yes - I think so, but that then also has sweeping changes, so I stopped here for now.